src/main/java/com/gregtechceu/gtceu/data/recipe/configurable/RecipeAddition.java

```diff
@@ -988,11 +1030,321 @@ private static void hardMiscRecipes(Consumer<FinishedRecipe> provider) {
+
+            ASSEMBLER_RECIPES.recipeBuilder("blast_furnace")
+                    .inputItems(plate, Iron, 3)
+                    .inputItems(new ItemStack(Blocks.FURNACE))
+                    .outputItems(new ItemStack(Blocks.BLAST_FURNACE))
+                    .duration(80).EUt(6).save(provider);
+
+            VanillaRecipeHelper.addShapedRecipe(provider, "composter", new ItemStack(Blocks.COMPOSTER), "PsP",
+                    "P P", "XPX",
+                    'P', ItemTags.PLANKS,
+                    'X', new UnificationEntry(screw, Wood));
+
+            ASSEMBLER_RECIPES.recipeBuilder("composter")
+                    .inputItems(ItemTags.PLANKS, 4)
+                    .circuitMeta(1)
+                    .outputItems(new ItemStack(Blocks.COMPOSTER))
+                    .duration(80).EUt(6).save(provider);
+
+            ASSEMBLER_RECIPES.recipeBuilder("bell")
+                    .inputItems(new ItemStack(Items.STICK))
+                    .inputItems(rod, Stone, 2)
+                    .inputItems(bolt, Iron, 2)
+                    .inputItems(plate, Bronze, 4)
+                    .outputItems(new ItemStack(Blocks.BELL))
+                    .duration(200).EUt(16).save(provider);
+
+            ASSEMBLER_RECIPES.recipeBuilder("conduit")
+                    .inputItems(new ItemStack(Items.HEART_OF_THE_SEA))
+                    .inputItems(new ItemStack(Items.NAUTILUS_SHELL, 8))
+                    .outputItems(new ItemStack(Blocks.CONDUIT))
+                    .duration(200).EUt(16).save(provider);
+
+            VanillaRecipeHelper.addShapedRecipe(provider, "candle", new ItemStack(Blocks.CANDLE), "r",
+                    "S", "W",
+                    'S', new ItemStack(Items.STRING),
+                    'W', new ItemStack(Items.HONEYCOMB));
+
+            ASSEMBLER_RECIPES.recipeBuilder("lodestone")
+                    .inputItems(plateDense, SteelMagnetic)
+                    .inputItems(screw, Iron, 4)
+                    .inputItems(plate, Stone, 4)
+                    .outputItems(new ItemStack(Blocks.LODESTONE))
+                    .duration(200).EUt(16).save(provider);
+
+            ASSEMBLER_RECIPES.recipeBuilder("music_disc_5")
+                    .inputItems(new ItemStack(Items.DISC_FRAGMENT_5, 9))
+                    .outputItems(new ItemStack(Items.MUSIC_DISC_5))
+                    .duration(100).EUt(6).save(provider);
+
+            VanillaRecipeHelper.addShapedRecipe(provider, "turtle_helmet", new ItemStack(Items.TURTLE_HELMET), "SSS",
+                    "SrS",
+                    'S', new ItemStack(Items.SCUTE));
+
+            VanillaRecipeHelper.addShapedRecipe(provider, "barrel", new ItemStack(Blocks.BARREL), "PSP",
+                    "PsP", "PSP",
+                    'P', ItemTags.PLANKS,
+                    'S', ItemTags.WOODEN_SLABS);
+
+            ASSEMBLER_RECIPES.recipeBuilder("barrel")
+                    .inputItems(ItemTags.PLANKS, 7) // needs circuit probably
```

Comment:
does it need a circuit?

Reply:
Tested, doesn't seem like it conflicts with anything